gips/data/landsat/landsat.py

```diff
@@ -312,11 +314,24 @@ def query_service(cls, asset, tile, date):
             )['data']
 
             for result in response['results']:
-                available.append({
-                    'basename': result['displayId'] + '.tar.gz',
-                    'sceneID': result['entityId'],
-                    'dataset': dataset,
-                })
+                metadata = requests.get(result['metadataUrl']).text
+                xml = ElementTree.fromstring(metadata)
+                # Indexing an Element instance returns it's children
+                scene_cloud_cover = xml.find(
+                    ".//{http://earthexplorer.usgs.gov/eemetadata.xsd}metadataField[@name='Scene Cloud Cover']"
+                )[0].text
+                land_cloud_cover = xml.find(
+                    ".//{http://earthexplorer.usgs.gov/eemetadata.xsd}metadataField[@name='Land Cloud Cover']"
+                )[0].text
+
+                if float(scene_cloud_cover) < 90.0:
```

Comment:
I think if we're going to filter, we should make it a parameter.  Something like `max_scene_cc` with `90` as a default.  Eventually, we need to wire the cloud filtering from the CLI/API to the fetch, so that we don't download junk scenes.

Reply:
Sure thing. How should it be paramaterized? Through an kwarg to `query_service`? Or in the user's settings file? Something else altogether? (Lot of question marks in this comment)

Reply:
My default is to add the parameter to the function, with a default so that calls which don't specify it aren't broken.  I don't think it should be a `settings` level thing, as the value used depends on how hungry the user is for data on a given day.  Are you thinking kwargs for how to wire filter parameters into the CLI/API?

Reply:
Oh, no, I guess I just meant parameter with a default. I always forget that kwargs is more about how it's called rather than how the function is defined. Anyway, that sounds good to me.